taichi/rhi/vulkan/vulkan_device.cpp

```diff
@@ -1750,6 +1751,7 @@ DeviceAllocation VulkanDevice::allocate_memory(const AllocParams &params) {
 
   VmaAllocationCreateInfo alloc_info{};
   if (export_sharing) {
+    alloc_info.flags |= VMA_ALLOCATION_CREATE_DEDICATED_MEMORY_BIT;
```

Comment:
[`VK_KHR_external_memory`](https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_external_memory.html) requires dedicated allocations for export images.